x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -31,7 +31,7 @@ export function registerIntegrations(
         src: core.http.basePath.prepend(`/plugins/${APP_ID}/assets/gdal_logo.svg`),
       },
     ],
-    categories: ['upload_file', 'geo'],
+    categories: ['enterprise_search'],
```

Comment:
Not all are moved to Enterprise Search, for example "Upload a file" is moved to Observability and Custom.

@serenachou @mukeshelastic Could you add some context here about why upload services like Elastic Maps Service are moved to Enterprise Search?

Reply:
I confirmed with Serena that `Enterprise Search` is definitely not the right category for these geo/maps integrations. Thanks for the sanity check @nreese.

I am discussing with Mukesh on what the right categories should be. I am leaning towards replacing the "Upload a file" category with "Custom" and bringing back the "Geo" category as there isn't really a suitable replacement amongst the trimmed-down list of top-level categories (maybe `Elastic Stack` would also work instead).

Reply:
@serenachou @jen-huang discussed it a little further and we would like to move  upload a file and geo related integrations to custom category.

Reply:
PR was updated to just have "Custom" for the file upload integration and geo integrations

Reply:
I am not sure users will be able to find them in custom. Would it be possible to keep the existing categories? I do not understand how this change is better/needed. Maybe some more background and context would help?

Reply:
I'll also add that the larger effort is to trim the amount of top-level categories presented when user enters Browse Integrations. Today the amount of categories is too long to be effective. With the changes in this PR and elsewhere, the list is much more streamlined: 

![image](https://user-images.githubusercontent.com/1965714/227374800-6fa27ea8-d78d-4037-8570-ce9f992280bc.png)

I am in the process of checking that searching on "geo" and "maps" will still bring up these integrations so that the user doesn't have to look in the Custom category. I will likely need to adjust the descriptions for these integrations so that they contain these keywords.

Reply:
Thanks @jen-huang for checking that geo and maps search will show these results

Reply:
Is there an opportunity to see an early build of this on snapshot? I wanted to check the 32 integrations that are associated with Enterprise Search

Reply:
@serenachou Not at the moment. Here's a screenshot of Enterprise Search from my local build, you can see the subcategories as well:
![image](https://user-images.githubusercontent.com/1965714/227616652-ee6eaa89-5e7d-4f5b-8687-ad9766abfa74.png)

(Ignore the weird cutoff of the right-hand column, issue with the screenshot tool)